internal/monitors/docker/metadata.yaml

```diff
@@ -14,292 +14,365 @@ monitors:
+        to the cgroup.
       included: false
       type: cumulative
+      group: memory
     memory.stats.pgpgout:
       description: Number of uncharging events to the memory cgroup. Uncharging events
         happen each time a page is unaccounted from the cgroup.
       included: false
       type: cumulative
+      group: memory
     memory.stats.rss:
-      description: 'The amount of memory that doesn’t correspond to anything on disk:
-        stacks, heaps, and anonymous memory maps.'
+      description: "The amount of memory that doesn\u2019t correspond to anything on\
+        \ disk: stacks, heaps, and anonymous memory maps."
       included: false
       type: gauge
+      group: memory
     memory.stats.rss_huge:
       description: Amount of memory due to anonymous transparent hugepages.
       included: false
       type: gauge
+      group: memory
     memory.stats.swap:
       description: Bytes of swap memory used by container
       included: false
       type: gauge
+      group: memory
     memory.stats.total_active_anon:
       description: Total amount of memory that has been identified as active by the
         kernel. Anonymous memory is memory that is not linked to disk pages.
       included: false
       type: gauge
+      group: memory
     memory.stats.total_active_file:
       description: Total amount of active file cache memory. Cache memory = active_file
         + inactive_file + tmpfs
       included: false
       type: gauge
+      group: memory
     memory.stats.total_cache:
       description: Total amount of memory used by the processes of this control group
         that can be associated with a block on a block device. Also accounts for memory
         used by tmpfs.
       included: false
       type: gauge
+      group: memory
     memory.stats.total_dirty:
       description: Total amount of memory waiting to get written to disk
       included: false
       type: gauge
+      group: memory
     memory.stats.total_inactive_anon:
-      description: Total amount of memory that has been identified as inactive by
-        the kernel. Anonymous memory is memory that is not linked to disk pages.
+      description: Total amount of memory that has been identified as inactive by the
+        kernel. Anonymous memory is memory that is not linked to disk pages.
       included: false
       type: gauge
+      group: memory
     memory.stats.total_inactive_file:
       description: Total amount of inactive file cache memory. Cache memory = active_file
         + inactive_file + tmpfs
       included: false
       type: gauge
+      group: memory
     memory.stats.total_mapped_file:
-      description: Total amount of memory mapped by the processes in the control group.
-        It doesn’t give you information about how much memory is used; it rather tells
-        you how it is used.
+      description: "Total amount of memory mapped by the processes in the control group.\
```

Comment:
Looks like a number of these got mangled. Shouldn't need escape sequences for apostrophe. Should probably convert them to using `>` if they're long enough.